### PR TITLE
Fix misleading EN Settings (2841)

### PR DESCRIPTION
**Pull request by steins-code**
fix misleading en settings as described in bug: https://jira.itc.sap.com/browse/EXPOSUREAPP-2841

How to reproduce these screens: 
Install new app -> During Onboarding first accept but then decline Bewegungsmitteilungen verwenden in OS consent (Click on "Abbrechen"). -> Go to EN Settings

IMG_4D408414DCF4-1
IMG_08F9B1BBF015-1

### CODE REVIEW

**Review by janetback — APPROVED** [review]
UA reviewed

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -783,9 +783,13 @@ Sie können Ihre Einwilligung jederzeit widerrufen, indem Sie die App löschen.
 
 "ExposureNotificationSetting_Detail_Action_Button" = "Geräte-Einstellungen öffnen";
 
-"ExposureNotificationSetting_Activate_OSENSetting" = "COVID-19-Kontaktprotokoll und -Mitteilungen aktivieren ";
+"ExposureNotificationSetting_Activate_OSENSetting" = "Begegnungsmitteilungen aktivieren";
 
-"ExposureNotificationSetting_Activate_OSENSetting_Description" = "Die Risiko-Ermittlung benötigt das aktivierte COVID-19-Kontaktprotokoll und die -Mitteilungen in den Systemeinstellungen, um Begegnungen zu protokollieren. Bitte aktivieren Sie „COVID-19-Kontaktprotokoll und -Mitteilungen“ in Ihren Systemeinstellungen.";
+"ExposureNotificationSetting_Activate_OSENSetting_Description" = "Die Risiko-Ermittlung benötigt, dass die Begegnungsmitteilungen in den Systemeinstellungen aktiviert sind, um Begegnungen zu protokollieren. Bitte klicken Sie auf „Geräte-Einstellungen öffnen“ -> „Begegnungsmitteilungen“ -> „Begegnungsmitteilungen teilen“. Legen Sie danach die App als aktive App für die Begegnungsmitteilungen fest, indem Sie auf „Als aktive Region festlegen“ klicken.";
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:788]: Hmm we still support 13.6 is the text changed here like this as well or do we need to provide different instructions for different OS-Versions?

File: src/xcode/ENA/ENA/Source/Models/States/ENStateHandler+State.swift [hunk 2]
```diff
@@ -16,6 +16,8 @@ extension ENStateHandler {
 		case restricted
 		///Not authorized. The user declined consent in onboarding.
 		case notAuthorized
+		///ENStatus is restricted but ENAuthorization Status is authorized. App was not set as active for the region.
+		case notActiveApp
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Models/States/ENStateHandler+State.swift:20]: Very nice find! 🤙

**Comment by marcussc** [comment]
Please add Label and Milestone :)

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:788]: good point, I will clarify

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -729,9 +729,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
+
+"ExposureNotificationSetting_Activate_Old_OSENSetting" = "COVID-19-Begegnungsmitteilungen aktivieren ";
+
+"ExposureNotificationSetting_Activate_Old_OSENSetting_Description" = "Die Risiko-Ermittlung benötigt aktivierte COVID-19-Begegnungsmitteilungen in den Systemeinstellungen, um Begegnungen zu protokollieren. Bitte klicken Sie auf „Geräte-Einstellungen öffnen“ und aktivieren dann die „COVID-19-Begegnungsmitteilungen“ in den Einstellungen.";
+
+"ExposureNotificationSetting_SetActiveApp_OSENSetting" = "Aktive Region festlegen";
+
+"ExposureNotificationSetting_SetActiveApp_Old_OSENSetting_Description" = "Die Risiko-Ermittlung benötigt, dass die Corona-Warn-App in den Systemeinstellungen als aktive App festgelegt wurde, um Begegnungen zu protokollieren. Bitte klicken Sie auf „Geräte-Einstellungen öffnen“ -> „Einstellungen für COVID-19-Begegnungsaufzeichnungen“ und wählen Sie dann die Corona-Warn-App als aktive App aus.";
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:742]: Nitpicking: 
Replacing Old with pre-13.7

**Review by marcussc — APPROVED** [review]

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**